src/content/en/updates/posts/2015/07/interact-with-ble-devices-on-the-web.markdown

```diff
@@ -270,9 +312,9 @@ peek of what to expect in the coming months:
   platform](https://github.com/WebBluetoothCG/web-bluetooth/issues/127) will be
   added to improve user experience.
 
-At the time of writing, Chrome OS and Android 6+ are [the most advanced
+At the time of writing, Chrome OS, Android 6+ and Chrome for Linux are [the most advanced
```

Comment:
Yes in https://github.com/WebBluetoothCG/web-bluetooth/blob/gh-pages/implementation-status.md
I didn't want to re-add it there to avoid duplication.

> Linux: Requires Kernel 3.19+ and BlueZ 5+ installed (by default in Ubuntu 15.10).
Note that Bluetooth daemon needs experimental interfaces feature enabled: sudo /usr/sbin/bluetoothd -E